src/meta/tags.js

```diff
@@ -34,6 +34,7 @@ module.exports = function(Meta) {
 					name: 'msapplication-square150x150logo',
 					content: Meta.config['brand:logo'] || ''
 				}];
+				if (link !== undefined) defaultTags.splice(4, 1);
```

Comment:
I don't believe leaving them on all pages would _harm_ SEO value... in either case, the correct course of action would be to add the `keywords` meta tag to `res.locals.metaTags` in `src/controllers/index.js`, `Controllers.home`. See how it is done in `src/controllers/topic.js`